pkg/metricsservice/api/metrics.pb.go

```diff
@@ -16,7 +16,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.21.12
+// 	protoc        v3.6.1
```

Comment:
please remove this from the PR, it is not relevant and we are downgrading the protoc version